src/software/full_system_main.cpp

```diff
@@ -121,6 +120,11 @@ int main(int argc, char **argv)
                 ->setValue(args.network_interface_name);
         }
 
+        if (args.backend_name.empty())
+        {
+            LOG(FATAL) << "The option '--backend' is required but missing";
+        }
+
```

Comment:
Curious why this change is required?

Reply:
not required, but it could be a better error message than `"" is not a recognized backend`, now that `--backend` is not a required argument